tests/bwc/test_upgrade.py

```diff
@@ -471,6 +480,13 @@ class SnapshotCompatibilityTest(NodeProvider, unittest.TestCase):
     VERSION = ('3.3.x', '4.x.x')
 
     def test_snapshot_compatibility(self):
+        try:
+            self.setUp()
```

Comment:
why is this needed? `setUp` and `tearDown` should be called by pythons unittest runner

Reply:
initially I added it just by following [example](https://github.com/crate/crate-qa/pull/216/commits/dc0c9fdd211b0b9a7bbad59a8c6d87f6320594de) from another test case

Reply:
ok, [code](https://github.com/crate/crate-qa/blob/master/tests/bwc/test_upgrade.py#L154-L160)  I taken as example explicitly calls setUp/tearDown as it runs this test for series of upgrade_paths and needs to refresh things in between - so I won't change it